services/web/client/tools/docker-compose.yml

```diff
@@ -12,7 +12,8 @@ services:
     working_dir: /project
     command: >
       qx serve --set qx.allowUrlSettings=true
-        --set osparc.vcsOriginUrl='"${VCS_URL}"'
-        --set osparc.vcsRef='"${VCS_REF}"'
-        --set osparc.vcsRefClient='"${VCS_REF_CLIENT}"'
-        --set osparc.vcsStatusClient='"${VCS_STATUS_CLIENT}"'
+        --set-env osparc.stackName="${SWARM_STACK_NAME}"
+        --set-env osparc.vcsOriginUrl="${VCS_URL}"
+        --set-env osparc.vcsRef="${VCS_REF}"
+        --set-env osparc.vcsRefClient="${VCS_REF_CLIENT}"
+        --set-env osparc.vcsStatusClient="${VCS_STATUS_CLIENT}"
```

Comment:
we could pass the build date as well it's ${BUILD_DATE}, see Makefile

Reply:
whatever you prefer.. (see above)

Reply:
I think this is also meaningful info, please add 👍